openvdb/points/AttributeArray.h

```diff
@@ -1105,6 +1115,12 @@ void AttributeArray::copyValues(const AttributeArray& sourceArray, const IterT&
     // if the target array is uniform, expand it first
     this->expand();
 
+    // TODO: Acquire mutex locks for source and target arrays to ensure that
+    // value copying is always thread-safe. Note that the unsafe method will be
+    // faster, but can only be used if neither the source or target arrays are
+    // modified during copying. Note that this will require a new private
+    // virtual method with ABI=7 to access the mutex from the derived class.
```

Comment:
perhaps ticket this up so we don't forget

Reply:
Done - https://jira.aswf.io/browse/OVDB-33